package.json

```diff
@@ -27,18 +26,18 @@
   },
   "homepage": "https://github.com/FAC-11/DataPower#readme",
   "devDependencies": {
-    "eslint": "^4.15.0",
+    "eslint": "^4.18.1",
     "eslint-config-airbnb": "^15.1.0",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.5.1",
+    "eslint-plugin-react": "^7.7.0",
     "istanbul": "^0.4.5",
-    "nodemon": "^1.14.11",
-    "nyc": "^11.4.1",
+    "nodemon": "^1.15.1",
+    "nyc": "^11.5.0",
```

Comment:
why do we have both `istanbul` _and_ `nyc`?

Reply:
ignore this, I've removed istanbul in #202

Reply:
Sweet I was about to ask